tests/cli-rustup.rs

```diff
@@ -834,6 +834,35 @@ fn show_toolchain_override_not_installed() {
     });
 }
 
+#[test]
+fn override_set_unset_with_path() {
+    setup(&|config| {
+        let workdir = config.current_dir();
+        let workdir = workdir.to_string_lossy();
+        config.change_dir(&config.emptydir, &|| {
+            expect_ok(
+                config,
+                &["rustup", "override", "set", "nightly", "--path", &workdir],
+            );
+        });
+        expect_ok_ex(
+            config,
+            &["rustup", "override", "list"],
+            &format!("{}\tnightly-x86_64-unknown-linux-gnu\n", &workdir),
```

Comment:
This line is making an assumption about the host platform which won't hold true on Windows or MacOS.

There is a macro `for_host!()` whose use you can see in other tests which will help you reformat this to be correct across the platforms.

Reply:
Thanks a lot, you got me looking, seems simple enough.